src/sage/schemes/curves/zariski_vankampen.py

```diff
@@ -634,15 +656,15 @@ def fieldI(field):
     if I0 in field:
         return field
     field_a = field[I0]
-    field_b = field_a.absolute_field('b0')
+    field_b = field_a.absolute_field('imaginary_unit')
     b0 = field_b.gen()
     q = b0.minpoly()
     qembd = field_b.embeddings(QQbar)
     for h1 in qembd:
         b1 = h1(b0)
         b2 = h1(field_b(field_a.gen(0)))
         b3 = field.gen(0)
-        F1 = NumberField(q, 'b', embedding=b1)
+        F1 = NumberField(q, 'primitif_element', embedding=b1)
```

Comment:
I don't agree with these changes. Yes, they are more descriptive, but they are also a lot more to parse and make the output harder to read. If you insist on changing them, I would make their names shorter.

Reply:
I agree, it was caused by stranged errors caused by the lack of choice of names in number fields, which has been resolved. I will change it.